Ui sprint4

PauloJaime opened the pull request.

pjaime88 committed:
US111-Task112: Added Menu For Printing Option

For this commit, I added a design menu  for printing inside the File Menu. Also, I added the shortcuts and made some code improvements

pjaime88 committed:
US111-Task113: Added The Code for Printing

For this commit, I added the code for menu print inside the File Menu. This will allow the user to send to a printer or save as pdf whatever he has on the current Tab. Also, I added some code improvements

pjaime88 committed:
US114-Task115: Added  The Text Alignment Option

For this commit, I added the design within the Font and Color Menu. This design consists in three buttons that will  allow the user to center, left or centralize his text. Also, I added some code improvements

pjaime88 committed:
US114-Task116: Added The Code for Text Alignment Option

For this commit, I added the code within the Font and Color Menu. This design consists in three buttons that will  allow the user to center, left or centralize his text. Also, I added some code improvements

pjaime88 committed:
US123-Task124: Added Python Option (Design)

For this commit, I added the python option within the Syntax Menu. This will allow the user not only to use Java format text or plain text but also python

pjaime88 committed:
US123-Task125: Added keywords for Python code highlighting

For this commit, I added two files that include the keywords for python to be used for text/code highlighting . This will allow the user to see his/her python code highlighted.

Weizitong reviewed (changes requested):
Hi Paulo, I find that there are some conflicts in the code. Please resolve them. Of you can pull the current Master branch and create a new branch and add your code to that branch. Then, you can create a new Pull Request

HFDiamond reviewed (commented):
I agree with Zitong. You need to either solve all the conflict or create new branch to add new features on. I will recommend the second one, because it will save you a lot of time.

Weizitong commented on src/main/resource/pythonkeyword.txt:
```diff
@@ -0,0 +1,41 @@
+raise
+
+
+//string and comment color
```
Python syntax doesn't use // as comment but '#'

pjaime88 committed:
US123-Task125: Added keywords for Python code highlighting(Fxing a bug)

For this commit I had to edit of the Keyword file.I had to specify that for python (#) is used for commets
